draft-ietf-moq-transport.md

```diff
@@ -742,76 +746,74 @@ done in the context of an established MoQT session.
 
-## Announcements
-
-A publisher MAY send ANNOUNCE messages to any subscriber. An ANNOUNCE indicates
-to the subscriber that the publisher has tracks available in that namespace. A
-subscriber MAY send SUBSCRIBE or FETCH for tracks in a namespace without having
-received an ANNOUNCE for it.
-
-If a publisher is authoritative for a given namespace, or is a relay that has
-received an authorized ANNOUNCE for that namespace from an upstream publisher,
-it MUST send an ANNOUNCE to any subscriber that has subscribed to ANNOUNCE for
-that namespace, or a more generic set including that namespace. A publisher MAY
-send the ANNOUNCE to any other subscriber.
-
-An endpoint SHOULD NOT, however, send an ANNOUNCE advertising a namespace that
-exactly matches a namespace for which the peer sent an earlier ANNOUNCE
-(i.e. an ANNOUNCE ought not to be echoed back to its sender).
-
-The receiver of an ANNOUNCE_OK or ANNOUNCE_ERROR SHOULD report this to the
-application to inform the search for additional subscribers for a namespace,
-or abandoning the attempt to publish under this namespace. This might be
-especially useful in upload or chat applications. A subscriber MUST send exactly
-one ANNOUNCE_OK or ANNOUNCE_ERROR in response to an ANNOUNCE. The publisher
-SHOULD close the session with a protocol error if it receives more than one.
-
-An UNANNOUNCE message withdraws a previous ANNOUNCE, although it is not a
-protocol error for the subscriber to send a SUBSCRIBE or FETCH message for a
-track in a namespace after receiving an UNANNOUNCE.
-
-A subscriber can send ANNOUNCE_CANCEL to revoke acceptance of an ANNOUNCE, for
-example due to expiration of authorization credentials. The message enables the
-publisher to ANNOUNCE again with refreshed authorization, or discard associated
-state. After receiving an ANNOUNCE_CANCEL, the publisher does not send
-UNANNOUNCE.
-
-While ANNOUNCE indicates to relays how to connect publishers and subscribers, it
+## Sending Publish Messages
+
+A non-relay publisher MAY send a PUBLISH message to any subscriber. PUBLISH
+indicates to the subscriber that the publisher has a single track available, or
+one or more tracks available in a namespace. A subscriber MAY send SUBSCRIBE or
+FETCH for a track without having received an PUBLISH for the tracks namespace.
+
+If a publisher is authoritative for a given track or namespace, or is a relay
+that has received an authorized PUBLISH for that namespace from an upstream
+publisher, it MUST send a PUBLISH to any subscriber that has subscribed to that
+namespace, or a more generic set including that namespace. A publisher MAY send
+the PUBLISH to any other subscriber, but a relay MUST NOT send a PUBLISH without
+an authorized SUBSCRIBE first.
+
+An endpoint SHOULD NOT, however, send a PUBLISH advertising a namespace or track
+that exactly matches a namespace for which the peer sent an earlier PUBLISH
+(i.e. a PUBLISH ought not to be echoed back to its sender).
```

Comment:
To clarify, this is within a session, correct?

Reply:
Yes